javav2/example_code/cloudfront/Readme.md

```diff
@@ -4,67 +4,173 @@ This README discusses how to run and test the Java code examples for Amazon Clou
 
 ## Running the Amazon CloudFront Java files
 
-The credential provider used in all code examples is ProfileCredentialsProvider. For more information, see [Using credentials](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html).
+The credential provider used in all code examples is ProfileCredentialsProvider. For more information,
+see [Using credentials](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html).
 
 **IMPORTANT**
 
-The Java examples perform AWS operations for the account and AWS Region for which you've specified credentials, and you may incur AWS service charges by running them. See the [AWS Pricing page](https://aws.amazon.com/pricing/) for details about the charges you can expect for a given service and operation.
-
-Some of these examples perform *destructive* operations on AWS resources, such as deleting a function. **Be very careful** when running an operation that deletes or modifies AWS resources in your account. It's best to create separate test-only resources when experimenting with these examples.
-
-To run these examples, you can setup your development environment to use Apache Maven or Gradle to configure and build AWS SDK for Java projects. For more information, 
-see [Get started with the AWS SDK for Java 2.x](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html). 
-
-
- ## Testing the Amazon CloudFront files
-
-You can test the Java code examples forAmazon CloudFront by running a test file named **CloudFrontTest**. This file uses JUnit 5 to run the JUnit tests and is located in the **src/test/java** folder. For more information, see [https://junit.org/junit5/](https://junit.org/junit5/).
-
-You can execute the JUnit tests from a Java IDE, such as IntelliJ, or from the command line by using Maven. As each test is ran, you can view messages that inform you if the various tests succeed or fail. For example, the following message informs you that Test 3 passed.
+The Java examples perform AWS operations for the account and AWS Region for which you've specified credentials, and you
+may incur AWS service charges by running them. See the [AWS Pricing page](https://aws.amazon.com/pricing/) for details
+about the charges you can expect for a given service and operation.
```

Comment:
The Java examples perform AWS operations for the account and AWS Region for which you've specified credentials, and you might incur AWS service charges by running them. For details about charges for a given service and operation, see the [AWS Pricing page](https://aws.amazon.com/pricing/).